lib/components/term.js

```diff
@@ -260,6 +269,7 @@ export default class Term extends React.PureComponent {
 
   focus() {
     this.term.focus();
+    document.getElementsByTagName('title')[0].text = this.term.title;
```

Comment:
Just use the assignment statement `document.title = newTitle;`.

Reply:
sure, done, didn't know about that one :)